core/domain/value_generators_domain.py

```diff
@@ -108,7 +126,18 @@ def get_generator_class_by_id(cls, generator_id):
         """Gets a generator class by its id.
 
         Refreshes once if the generator is not found; subsequently, throws an
-        error."""
+        error.
+
+        Args:
+            generator_id: An id corresponding to a generator class.
+
+        Returns:
+            A generator class mapping to the generator id given.
+
+        Throws:
+            KeyError: An error occured during accessing
+            cls.value_generators_dict[generator_id]
```

Comment:
Indent by 4.

But actually, a better description would be something like: "The given generator_id is invalid."